### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by jairovadillo**
Adding integration tests using GoAWS to simulate SNS+SQS.

### CODE REVIEW

File: integration_tests/sqs_consumer_tests.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+from molange.consumers import MessageConsumer
+from molange.drivers import get_sqs_driver, Message
+from molange.event_manager import event_manager
+from molange.publisher import get_sns_message_publisher
+from molange.subscribers import Subscriber
+
+
+# events.py
+class FollowCreatedEvent:
```
Comment (guillem21b) [inline integration_tests/sqs_consumer_tests.py]: dataclass pels events estaria pepi, no?

File: README.md [hunk 2]
```diff
@@ -4,87 +4,32 @@
...
-## Versioning
-
-We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags). 
-
 ## Authors
 
-* **Billie Thompson** - *Initial work* - [PurpleBooth](https://github.com/PurpleBooth)
-
-See also the list of [contributors](https://github.com/your/project/contributors) who participated in this project.
+* **Jairo Vadillo** - *Initial work* - [JairoVadillo](https://github.com/jairovadillo)
```
[the remark this answers is not in the document]
Comment (jairovadillo) [reply README.md]: hahah Billy el Niño deixaré

File: integration_tests/sqs_consumer_tests.py (hunk 1 above)
Comment (jairovadillo) [reply integration_tests/sqs_consumer_tests.py]: ho vaig pensar pero aleshores estem forçats a tenir python3.7 no?

**Comment by jairovadillo** [comment]
New PR opened: #11

**Pull request closed without merging**